### DISCUSSION TITLE
Draw.io Mobile App (iOS) & Sketchpads

### DISCUSSION THREAD

**Discussion by Jay-Sal** [jgraph/drawio-desktop#1610] [General]
Hi, All - 
Just getting into this app and its awesome! I love that it integrates with Google suite.
I'm hoping someone might be able to help with understanding the sketchpad. I'm trying to create sketchpads items via desktop to save that way when using the mobile version of the app it's easy to map diagrams with our specific objects. But the mobile app Sketchpad does not copy the items from desktop login to mobile app login...

Am I doing something wrong...? There must be a way to sync this with the desk...?

Thank you all in advance!
- Jared

**Comment by davidjgraph** [comment]
There is no concept of login on the desktop, it's fully isolated by design.
